src/Dfe.PlanTech.AzureFunctions/Functions/QueueReceiver.cs

```diff
@@ -87,6 +92,45 @@ private async Task ProcessMessage(ServiceBusReceivedMessage message, ServiceBusM
         }
     }
 
+    private bool ShouldDropMessage(CmsEvent cmsEvent)
+    {
+        if (cmsEvent == CmsEvent.CREATE)
```

Comment:
Probably worth merging this if statement in with the `ShouldIgnoreMessage` method.

Otherwise we've got:

`ShouldDropMessage` calling `ShouldIgnoreMessage`, whilst also doing another check for a separate event first. Might be confusing

Reply:
Resolved in: [960e639](https://github.com/DFE-Digital/plan-technology-for-your-school/pull/508/commits/960e63987bce7721bde6170b944b12ebcdca5d03)